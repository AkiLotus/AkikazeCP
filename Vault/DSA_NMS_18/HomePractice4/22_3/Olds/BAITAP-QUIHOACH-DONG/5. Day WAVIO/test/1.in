12 
-5 10 0 5 10 9 15 -5  5 10 1 15

