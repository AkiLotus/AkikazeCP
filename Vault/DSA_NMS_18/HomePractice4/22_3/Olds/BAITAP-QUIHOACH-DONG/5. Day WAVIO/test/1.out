7
-5 0 5 10 9 5 1 